.github/workflows/build.yml

```diff
@@ -7,6 +7,9 @@ on:
       runner:
         required: true
         type: string
+      runner_for_virt:
```

Comment:
nit: can we name this `runner_with_kvm` instead? Since that's what we care about: that they have kernel accelerated virtualization through KVM (as you mention in the body, the other VMs did have virt, they were just obnoxiously slow compared to a KVM-having runner)

Reply:
well... I started with it called `kvm`, but changed it to virt because for example it isn't kvm for macOS.